Assets/Src/Scripts/Input/WebCamSource.cs

```diff
@@ -0,0 +1,203 @@
+          _width = 1600;
+          _height = 900;
+          break;
+        case (int)ResolutionOptions.HIGH:
+          _width = 1920;
+          _height = 1080;
+          break;
+        default:
+          break;
+      }
+    }
+
+    // DropdownChange event.
+    public void FpsDropdownChange(int value) {
+      switch (value) {
+        case (int)FpsOptions.LOW:
+          Fps = 60;
+          break;
+        case (int)FpsOptions.HiGH:
+          Fps = 30;
+          break;
+        default:
+          break;
+      }
+    }
+
+    // DropdownChange event.
+    public void FlipDropdownChange(int value) {
+      switch (value) {
+        case (int)FlipOptions.NORMAL:
+          Flip = false;
+          break;
+        case (int)FlipOptions.MIRRORING:
+          Flip = true;
+          break;
+        default:
+          break;
+      }
+    }
+
+    private void InitDropdown(Dropdown target, List<string> optarr, DropdownKey dropdownkey) {
+      target.ClearOptions();
+      switch (dropdownkey) {
+        case DropdownKey.CAMERA:
+          target.onValueChanged.AddListener(CameraDropdownChange);
+          break;
+        case DropdownKey.RESOLUTION:
+          foreach (ResolutionOptions opt in Enum.GetValues(typeof(ResolutionOptions))) {
+            optarr.Add(_resolutionDic[opt]);
+          }
+          target.onValueChanged.AddListener(ResolutionDropdownChange);
+          break;
+        case DropdownKey.FPS:
+          foreach (FpsOptions opt in Enum.GetValues(typeof(FpsOptions))) {
+            optarr.Add(_fpsDic[opt]);
+          }
+          target.onValueChanged.AddListener(FpsDropdownChange);
+          break;
+        case DropdownKey.FLIP:
+          foreach (FlipOptions opt in Enum.GetValues(typeof(FlipOptions))) {
+            optarr.Add(_flipDic[opt]);
+          }
+          target.onValueChanged.AddListener(FlipDropdownChange);
+          break;
+        default:
+          break;
+      }
+      target.AddOptions(optarr);
+      target.value = 0;
+      target.captionText.text = target.options[0].text;
+    }
+
+    // Webcam params init.
+    IEnumerator Start() {
+      yield return Application.RequestUserAuthorization(UserAuthorization.WebCam);
+      if (Application.HasUserAuthorization(UserAuthorization.WebCam)) {
+        WebCamDevice[] _devices = WebCamTexture.devices;
+        foreach (var device in _devices) {
+          _devicesName.Add(device.name);
+        }
+        InitDropdown(CameraDropdown, _devicesName, DropdownKey.CAMERA);
+      }
+      InitDropdown(ResolutionDropdown, _resolutionOpt, DropdownKey.RESOLUTION);
+      InitDropdown(FpsDropdown, _fpsOpt, DropdownKey.FPS);
+      InitDropdown(FlipDropdown, _flipOpt, DropdownKey.FLIP);
+    }
+
+    void Update() {
+      if (_webCam != null) {
+        if (!_webCam.didUpdateThisFrame) return;
+        bool vFlip = _webCam.videoVerticallyMirrored;
+        Vector2 scale = new Vector2(Flip ? -1 : 1, vFlip ? -1 : 1);
```

Comment:
```suggestion
        var scale = new Vector2(Flip ? -1 : 1, vFlip ? -1 : 1);
```

This is optional - google style: Use of var is encouraged if it aids readability by avoiding type names that are noisy, obvious, or unimportant.